### PR TITLE
BaSP-TG-62: get and delete admins with redux

### PR DESCRIPTION
**Pull request by Licamus**
get and delete admins with redux

### CODE REVIEW

File: src/Components/Shared/Table/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styles from './table.module.css';
-import Delete from '../../../assets/trash.png';
+import Delete from '../../../Assets/close_cross.png';
```
Comment (lmalvasia) [inline src/Components/Shared/Table/index.js:3]:
```suggestion
import Delete from '../../../assets/close_cross.png';
```

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -43,6 +35,7 @@ const Admins = () => {
     <>
       <div className={styles.container}>
         <h1>Admins</h1>
+        <Spinner isLoading={isLoading} />
```
Comment (lmalvasia) [inline src/Components/Admins/index.js:38]: When isLoading is true, we have to render only this component:
```
  if (isLoading) {
    return <Spinner isLoading={isLoading} />;
  }
  ```

Comment (Licamus) [reply src/Components/Admins/index.js:38]: done

File: src/Components/Shared/Table/index.js (hunk 1 above)
Comment (Licamus) [reply src/Components/Shared/Table/index.js:3]: done!

**Review by lmalvasia — APPROVED** [review]

File: src/redux/admins/actions.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import {
+  GET_ADMINS_PENDING,
+  GET_ADMINS_SUCCESS,
+  GET_ADMINS_ERROR,
+  DELETE_ADMINS_PENDING,
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:5]:
```suggestion
  DELETE_ADMIN_PENDING,
```

File: src/redux/admins/actions.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  DELETE_ADMINS_SUCCESS,
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:6]:
```suggestion
  DELETE_ADMIN_SUCCESS,
```

File: src/redux/admins/actions.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  DELETE_ADMINS_ERROR
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:7]:
```suggestion
  DELETE_ADMIN_ERROR
```

File: src/redux/admins/actions.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+export const getAdminsError = (error) => {
+  return {
+    type: GET_ADMINS_ERROR,
+    payload: error
+  };
+};
+
+export const deleteAdminsPending = () => {
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:30]:
```suggestion
export const deleteAdminPending = () => {
```

File: src/redux/admins/actions.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    type: DELETE_ADMINS_PENDING
+  };
+};
+
+export const deleteAdminsSuccess = (data) => {
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:36]:
```suggestion
export const deleteAdminSuccess = (data) => {
```

File: src/redux/admins/actions.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  return {
+    type: DELETE_ADMINS_SUCCESS,
+    payload: data
+  };
+};
+
+export const deleteAdminsError = (error) => {
```
Comment (baudraccoverona) [inline src/redux/admins/actions.js:43]:
```suggestion
export const deleteAdminError = (error) => {
```

File: src/redux/admins/constants.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+export const GET_ADMINS_PENDING = 'GET_ADMINS_PENDING';
+export const GET_ADMINS_SUCCESS = 'GET_ADMINS_SUCCESS';
+export const GET_ADMINS_ERROR = 'GET_ADMINS_ERROR';
+
+export const DELETE_ADMINS_PENDING = 'DELETE_ADMINS_PENDING';
```
Comment (baudraccoverona) [inline src/redux/admins/constants.js:5]: Same as above for all the delete constants

**Pull request merged**